config/applications.json

```diff
@@ -535,6 +535,14 @@
 		"link": "https://www.mozilla.org/en-US/firefox/new/",
 		"winget": "Mozilla.Firefox"
 	},
+	"WPFInstallfirefoxesr": {
+		"category": "Browsers",
+		"choco": "FirefoxESR",
+		"content": "Firefox ESR",
+		"description": "Mozilla Firefox is an open-source web browser known for its customization options, privacy features, and extensions.",
```

Comment:
I would put that ESR is the Extended Support Release that is updated approximately [every 42 weeks with minor updates.](https://support.mozilla.org/en-US/kb/choosing-firefox-update-channel)

Reply:
Updated the description in a new commit
Thanks for pointing this out @Today20092 😄